# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1264577057 18000
#      Wed Jan 27 02:24:17 2010 -0500
# Node ID eea99d24adae1c0c315f5631280820feb5e6cb19
# Parent  1d430a849f3c50e407eea5a1bc85ca6276885b76
update doc/interpreter/contributors.in

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -205,16 +205,17 @@ Julius Smith
 Shan G. Smith
 Joerg Specht
 Quentin H. Spencer
 Christoph Spiel
 Richard Stallman
 Russell Standish
 Doug Stewart
 Jonathan Stickel
+Judd Storrs
 Thomas Stuart
 Ivan Sutoris
 John Swensen
 Ariel Tankus
 Georg Thimm
 Duncan Temple Lang
 Kris Thielemans
 Olaf Till
